src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="big-bar"
```

Comment:
![image](https://user-images.githubusercontent.com/98764641/205622541-5e0eba90-92ac-4ab8-afcd-a74d519f38f0.png)
Do the same with forms and imgs